rskj-core/src/main/java/co/rsk/pcc/ExecutionEnvironment.java

```diff
@@ -51,7 +51,9 @@ public ExecutionEnvironment(
             BlockStore blockStore,
             ReceiptStore receiptStore,
             List<LogInfo> logs) {
-        this.activations = activationConfig.forBlock(block.getNumber());
+        // activations may be set to null here because some NativeContracts may not need
+        // a block number. An example of this is the Environment contract.
+        this.activations = block == null ? null : activationConfig.forBlock(block.getNumber());
```

Comment:
why do you think that Environment contract doesn't need an activation block height?